wiki/mapping/glossary.md

```diff
@@ -89,6 +89,7 @@ let us know in #mapping-discussion!
 | **First Person Flying Controller** | A launch option available to SteamVR and Oculus users to launch Beat Saber with limited keyboard and mouse control over the game in a first person view. Very helpful for getting an accurate preview of [Lighting](#l) or [Walls](#wxyz), or accessing the official [Editor](#e). Abbreviated as [FPFC](#f). |
 | **Fixed BPM** | A song with a consistent [BPM](#b) from start to finish with no variation. Also known as [Single BPM](#s). See also: [Variable BPM](#uv). |
 | **Flick** | A [Pattern](#p) of two or more [Notes](#n) of the same color, typically at 1/4 [Precision](#p). <details><summary>**Example Image**</summary>![Picture of flick](~@images/mapping/flick-alt.png)</details> |
+| **Flower** | A combination of two or more [Notes](#n) that are different directions, creating a flower like shape. 
```

Comment:
Could you provide an image containing a few examples in editor?

Reply:
Sure, attatch it here or in the repo?

Reply:
either is fine

Reply:
Alright, Sorry it took me so long, GitHub did not want to cooperate.
![Screenshot (8)](https://user-images.githubusercontent.com/72290220/125217368-71f44780-e28e-11eb-8870-131835157f08.png)